reverse_engineering/helpers/queryConstants.js

```diff
@@ -183,6 +183,215 @@ const queryConstants = {
         LEFT JOIN pg_catalog.pg_description AS pg_description ON (pg_description.objsubid=pg_attribute.attnum
                                                                AND pg_description.objoid = pg_attribute.attrelid)
         WHERE pg_attribute.attrelid = $1;`,
+	GET_TABLE_COLUMNS_FROM_CATALOG: `
```

Comment:
It looks like this one is from the view select statement, could you please add a comment leading to the original view. Like it is done in `GET_TRIGGERS`

```
-- This is information_schema.triggers view but without permitions
```